src/snowflake/snowpark/table.py

```diff
@@ -655,3 +661,11 @@ def merge(
             if block
             else result
         )
+
+    def drop_table(self) -> None:
```

Comment:
What about adding an optional parameter `if_exist`?

Reply:
Very good suggestion. Updated.

Reply:
I was thinking of having an optional parameter in the method `drop_table` and if that is set to 
 true, we append `if exists` to the end of the sql. Maybe the user wants an operation to fail if the table has been dropped/not created properly before.

Reply:
As offline discussion, we'll not add the param and add it later when we think about other Snowflake objects.